.github/workflows/unit_tests.yml

```diff
@@ -71,8 +71,8 @@ jobs:
       matrix:
         module: [ ctriface, ctriface/image, devmapper ]
     steps:
-    
-    - name: Set up Go 1.19
+
```

Comment:
This line break the unit tests for some reason (https://github.com/vhive-serverless/vHive/actions/runs/6322600095/workflow#L74)

Reply:
Fixed.